tools/cldr-code/src/main/java/org/unicode/cldr/util/Containment.java

```diff
@@ -203,16 +204,41 @@ public Set<String> getSubontinents() {
 
     public static Set<List<String>> getAllDirected(Multimap<String, String> multimap, String lang) {
         LinkedHashSet<List<String>> result = new LinkedHashSet<>();
-        getAllDirected(multimap, lang, new ArrayList<String>(), result);
+        getAllDirected(
+                multimap,
+                lang,
+                new ArrayList<String>(),
+                result,
+                new LinkedList<String>(),
+                new HashSet<String>());
         return result;
     }
 
     private static void getAllDirected(
             Multimap<String, String> multimap,
             String lang,
             ArrayList<String> target,
-            Set<List<String>> targets) {
-        target.add(lang);
+            Set<List<String>> targets,
+            List<String> depth,
```

Comment:
"depth" sounds like an integer. Maybe "ancestors"?

Reply:
That's a good point.  It was originally an integer but I changed it when I needed more than just the depth.  Will update.